_examples/wrapper/python/wrapper_code.go

```diff
@@ -0,0 +1,24 @@
+// Copyright 2015 The go-python Authors.  All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package wrapper_python
```

Comment:
as you named the package `wrapper_python`, please rename the directory holding that file to `wrapper_python`.
(or `pywrap` ? `pywrapper` ?)